### ISSUE TITLE
Issue with docs/tasks/traffic-management/circuit-breaking/index.md

### ISSUE THREAD

**Issue by phanibalaji** [istio/istio.io#2254]
While creating the destination rule below , we need to specify the tls traffic policy as ISTIO_MUTUAL if mTLS is enabled in ISTIO env. 

```
cat <<EOF | kubectl apply -f -
apiVersion: networking.istio.io/v1alpha3
kind: DestinationRule
metadata:
  name: httpbin
spec:
  host: httpbin
  trafficPolicy:
    connectionPool:
      tcp:
        maxConnections: 1
      http:
        http1MaxPendingRequests: 1
        maxRequestsPerConnection: 1
    outlierDetection:
      consecutiveErrors: 1
      interval: 1s
      baseEjectionTime: 3m
      maxEjectionPercent: 100
EOF
```

The above should be 

```
cat <<EOF | kubectl apply -f -
apiVersion: networking.istio.io/v1alpha3
kind: DestinationRule
metadata:
  name: httpbin
spec:
  host: httpbin
  trafficPolicy:
    **tls:
      mode: ISTIO_MUTUAL**
    connectionPool:
      tcp:
        maxConnections: 1
      http:
        http1MaxPendingRequests: 1
        maxRequestsPerConnection: 1
    outlierDetection:
      consecutiveErrors: 1
      interval: 1s
      baseEjectionTime: 3m
      maxEjectionPercent: 100
EOF
```
Otherwise the calls to httpbin service fails in the subsequent step with http 503 error. Do we need to mention this in the doc??

**Comment by frankbu** [comment]
Yes, thanks for pointing it out @phanibalaji.

I think the mirroring task also has this issue.

**Comment by frankbu** [comment]
This issue has been fixed in the 1.1 docs by adding a note in both the circuit breaking and mirroring tasks.
